### DISCUSSION TITLE
填入rrocr的appkey后无法过验证码，且在后台查询没有识别记录

### DISCUSSION THREAD

**Discussion by Shiliang-vector** [mikumifa/biliTickerBuy#134] [提问专区]
选择使用rrocr方式后启动过验证码测试，一真卡在图一页面，手动点按钮验证是可以通过的，在rrocr后台查不到过验证码的记录，是我使用的方式有问题吗？
![测试1](https://github.com/mikumifa/biliTickerBuy/assets/137939685/e7a3ef74-3680-4a7c-b92e-b82ed32b7b72)
![测试2](https://github.com/mikumifa/biliTickerBuy/assets/137939685/bd967ddb-58e5-43a4-a525-ddbdfe95fa54)

**Comment by Shiliang-vector** [comment]
看到已经有人提交议题了

**Comment by mikumifa** [comment]
被我改坏了，下次迭代修，现在更新东西太少了，攒多点发过去

**Reply by Shiliang-vector** [reply]
辛苦了😘😘😘
